docs/launchpad/guides/kubernetes-guide.md

```diff
@@ -203,26 +206,289 @@ To add more control-plane nodes you need to first get the hosts ready for such b
+```
+
+**5:** Set variables for CRI-O commands:
+```bash
+export OS="xUbuntu_22.04"
+export VERSION="1.28"
+```
+
+**6:** Install CRI-O:
+```bash
+echo "deb [signed-by=/usr/share/keyrings/libcontainers-archive-keyring.gpg] https://download.opensuse.org/repositories/devel:/kubic:/libcontainers:/stable/$OS/ /" > /etc/apt/sources.list.d/devel:kubic:libcontainers:stable.list
+echo "deb [signed-by=/usr/share/keyrings/libcontainers-crio-archive-keyring.gpg] https://download.opensuse.org/repositories/devel:/kubic:/libcontainers:/stable:/cri-o:/$VERSION/$OS/ /" > /etc/apt/sources.list.d/devel:kubic:libcontainers:stable:cri-o:$VERSION.list
+
+mkdir -p /usr/share/keyrings
+curl -L https://download.opensuse.org/repositories/devel:/kubic:/libcontainers:/stable/$OS/Release.key | gpg --dearmor -o /usr/share/keyrings/libcontainers-archive-keyring.gpg
+curl -L https://download.opensuse.org/repositories/devel:/kubic:/libcontainers:/stable:/cri-o:/$VERSION/$OS/Release.key | gpg --dearmor -o /usr/share/keyrings/libcontainers-crio-archive-keyring.gpg
+
+apt-get update
+apt-get install cri-o cri-o-runc
+
+systemctl daemon-reload
+systemctl enable --now crio
+```
+
+**7:** Install kubernetes pacakges:
+```bash
+curl -fsSL https://pkgs.k8s.io/core:/stable:/v${VERSION}/deb/Release.key | sudo gpg --dearmor -o /etc/apt/keyrings/kubernetes-apt-keyring.gpg
+echo "deb [signed-by=/etc/apt/keyrings/kubernetes-apt-keyring.gpg] https://pkgs.k8s.io/core:/stable:/v${VERSION}/deb/ /" | sudo tee /etc/apt/sources.list.d/kubernetes.list
+apt-get update
+apt-get install -y kubelet kubeadm kubectl
+```
+
+**8:** Hold package versions so they don't auto-update:
+```bash
+apt-mark hold kubelet kubeadm kubectl
+```
+
+### Initialize the Cluster
+
+**9:** Create a kubeadm config for initializing the Cluster:
+```bash
+cat << EOF > /tmp/cluster-config.yaml
+apiVersion: kubeadm.k8s.io/v1beta3
+kind: InitConfiguration
+nodeRegistration:
+  kubeletExtraArgs:
+    cgroup-driver: systemd
+    node-ip: 10.110.0.2
+  taints: []
+skipPhases:
+  - addon/kube-proxy
+---
+apiVersion: kubeadm.k8s.io/v1beta3
+kind: ClusterConfiguration
+networking:
+  serviceSubnet: "10.96.0.0/16"
+  podSubnet: "10.10.0.0/16"
+controllerManager:
+  extraArgs:
+    allocate-node-cidrs: "true"
+    node-cidr-mask-size: "20"
+kubernetesVersion: "v1.28.3"
+controlPlaneEndpoint: 10.110.0.2 
+EOF
+```
+
+***Note***: If you intend to setup a HA Cluster, you should take care of setting up the VIP beforehand (be it by creating a Load Balancer in a Cloud Provider, or using a bare-metal solution based on something like Keepalived). That VIP (or DNS) should go into the `controlPlaneEndpoint`, as changing this after creating the Cluster is an elaborate endeavor.
+
+Here, we are also passing along a specific node-ip to guarantee the internal interface is gonna be used as the node we're using has multiple interfaces/IPs, and we're skipping kube-proxy install because we intend to use cilium CNI replacing kube-proxy.
```

Comment:
```suggestion
We are specifying a particular node-IP to ensure usage of the internal interface, as our node has multiple interfaces/IPs. We are also skipping the kube-proxy installation because we plan to use Cilium CNI, which will replace kube-proxy.
```